gitgud/__main__.py

```diff
@@ -30,14 +30,38 @@ def __init__(self):
 
         self.parser = argparse.ArgumentParser(prog='git gud')
 
-        load_description = \
-            "Load a specific skill or level. This command can be used in several ways." \
-            "" \
-            "Examples include:" \
-            "" \
-            "" \
-            "All of these load the same level"
-
+        load_description = '\n'.join([
+            "Load a specific skill or level. This command can be used in several ways.",
+            "\n",
+            "============Basic Usage\n",
+            "These two commands are the simplest commands to load a level on a certain skill, and are completely identical in functionality:\n",
+            "   git gud load <skill> <level>",
+            "   git gud load <skill>-<level>\n",
+            "<skill> and <level> could either be the actual name of the skill/level, or it can be the number of the skill/level.",
+            "Running `git gud skills` will help you find the number and name associated with each skill/level.\n",
+
+            "Here are a few example uses, which all load the same level:\n",
+            "   git gud load basics branching",
+            "   git gud load basics-2",
+            "   git gud load 1 branching",
+            "\n",
+            "============Additional Commands\n",
+            "`git gud load` supports additional shortcut commands that may make it easier for you to navigate through levels and skills.\n",
+            "======Loading a level on the current skill\n",
+            "This command loads the specified level of the current skill.\n",
+            "   git gud load -<level>\n",
+            "<level> MUST be the number of the level in order for this command to work.",
+            "Here is an example use, which loads the third level on the current skill:\n",
+            "   git gud load -3",
+            "\n",
+            "======Loading the first level on a skill\n",
+            "This command loads the first level on the specified skill:\n",
+            "   git gud load <skill>\n",
+            "<skill> may be either the name or the number of the skill.",
+            "Here are some example uses, which both load the first level of the first skill:\n",
+            "   git gud load basics",
+            "   git gud load 1",
+        ])
```

Comment:
This might be too long, especially when looking at it all on the command line. Do you think you could get it down to 15-20 lines?

You seem to be restating things towards the end. It's good to be clear, but at that point we're sacrificing usability for clarity. This is so long that users would have to scroll for a bit to get back to where they were. People should be able to figure it out with just a few examples. 

The shorter the better, just make sure it's understandable